Automattic/wp-calypso
# Help Center: add drag icon to header

Issue was listed [here](https://docs.google.com/spreadsheets/d/1URiJTt4CC5bC_7-m3UolwrhcJP9ZbaLB7uC-AWC5DaI/edit?gid=1972884515#gid=1972884515&range=41:41)

## Proposed Changes

Add drag icon and refactor header component.

## Why are these changes being made?

It is still a bit unclear that the Help Center can be dragged around. This icon should help indicate that the box itself is able to be dragged. While adding another piece to the header that was conditional based on `isMinimized` it made the component a little complicated this led me to create two separate contents. One for minimized and another for open.

| Before | After |
| - | - |
| <img width="858" alt="Markup 2024-07-15 at 21 44 39" src="https://github.com/user-attachments/assets/bb9affd8-0e3b-41ba-84c3-0559cdc5220d"> | <img width="852" alt="Markup 2024-07-15 at 21 43 54" src="https://github.com/user-attachments/assets/a9250526-4bbf-41e5-9e12-04602b688f3f"> |
| <img width="858" alt="Markup 2024-07-15 at 21 44 12" src="https://github.com/user-attachments/assets/53e2652f-5bad-47a6-b32e-cb879bfbbdc6"> | <img width="858" alt="Markup 2024-07-15 at 21 44 12" src="https://github.com/user-attachments/assets/53e2652f-5bad-47a6-b32e-cb879bfbbdc6"> |

## Testing Instructions

Open Calypso Live Link
Open Help Center to make sure it looks ok.
This is a simple style change so there shouldn't be any differences across locations.
